vic/drivers/image/src/vic_force.c

```diff
@@ -220,6 +321,17 @@ vic_force(void)
                                                 param.SNOW_MAX_SNOW_TEMP,
                                                 &(atmos[i].prec[j]), 1);
         }
+        // Validate vegcover
+        for (v = 0; v < options.NVEGTYPES; v++) {
+            vidx = veg_con_map[i].vidx[v];
+            if (vidx != -1) {
+                for (j = 0; j < NF; j++) {
+                    if (veg_hist[i][vidx].vegcover[j] < MIN_VEGCOVER) {
+                        veg_hist[i][vidx].vegcover[j] = MIN_VEGCOVER;
+                    }
+                }
+            }
+        }
```

Comment:
So, I did add code to raise a warning, and pushed it to origin at least 1 day ago, but it's not showing up here...